pipeline/pipeline.py

```diff
@@ -0,0 +1,38 @@
+"""File to combine the extract and loading into RDS and S3 scripts, to run the pipeline in a lambda function."""
+import logging
+from dotenv import load_dotenv
+
+from extract import extract
+from load import load
+from rds_to_s3 import update_rds_and_s3
+
+
+def set_up_logger():
+    """Set up a logger, to log pipeline progress to the console"""
```

Comment:
Add full stop to doc string to keep consistent with the others.

Reply:
yep just changed them, thanks!